apps/css-workshop/src/pages/tile.astro

```diff
@@ -1,6 +1,7 @@
 ---
 import Layout from './_layout.astro';
 import Badge from '../components/Badge.astro';
+import Tag_ from '../components/Tag.astro';
```

Comment:
Is there a reason that the tag component is imported as `Tag_` while the badge component is imported as just `Badge`? It feels like all Astro components should use the same conventions when possible.

Reply:
I think that's a mistake with the `Badge` component which I didn't work on. It was my understanding that we landed on the underscore https://github.com/iTwin/iTwinUI/pull/1948#discussion_r1538102242

Reply:
It seems like that was only done in the `badge.astro` file because the name `Badge` caused a conflict. It doesn't here in the `tile.astro` file, so there is no reason to use `_` to differentiate the name. The same is true for the `Tag` component here, so it seems to me like it also shouldn't have the `_`.

Reply:
Actually, rereading it, I think you might be right. In that case, the `Badge` here should be `Badge_`.

Reply:
Yeah I get the same error Mayank did when trying to remove it. I'll add the underscore to `Badge`

Reply:
ok yeah i'll move that to a new pr